packages/transformer/src/IModelExporter.ts

```diff
@@ -19,6 +19,7 @@ import {
   ElementMultiAspect,
   ElementRefersToElements,
```

Comment:
**Requirements from my side:**  
- Possibility to add custom changes (SourceDbChanges). 
- Custom changes should not be overridden by changes in changesets. 
- It is preferable to add changes after those are initialized from changesets. This will provide possibility to add custom changes based on changes in changesets.
- It should be possible to run processChanges transformation with custom changes when there are no changes in source. This is least priority (although still important) and will be tackled in separate PR:  https://github.com/iTwin/imodel-transformer/issues/170

Reply:
Regarding this requirement: "Custom changes should not be overridden by changes in changesets."

Is this still a requirement if you will be adding custom changes after changes in changesets are processed and you're using the sourceDbChanges as guidance for what to add? 

I think its possible to do custom changes both before and after at the moment, so maybe we should either look to make it only after or make it so that ones are added before are the single source of truth? I suppose I lean towards only after changesets are processed if thats possible, haven't looked into it yet.